src/main/shell-session/shell-env-modifier/shell-env-modifier-registration.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+
+import type { noop } from "../../../common/utils";
+
+export interface ShellEnvClusterData {
```

Comment:
For future proof reasons this could be just `ShellEnvContext` (terminal can be also in other contexts within Lens). It would have just one property, CatalogEntity.

Reply:
That is probably a good idea because the other info should be present if it is a KubernetesCluster